nixos/modules/services/monitoring/cadvisor.nix

```diff
@@ -112,6 +122,7 @@ in {
             -logtostderr=true \
             -listen_ip="${cfg.listenAddress}" \
             -port="${toString cfg.port}" \
+            ${concatStringsSep " " cfg.extraOptions} \
```

Comment:
You should use `lib.escapeShellArgs` here to make sure everything gets quoted correctly (I think even including multiline stuff)